### ISSUE TITLE
TLS Deprecation in JDK11.0.11_9

### ISSUE THREAD

**Issue by Bryksin** [liquibase/docker#71]
Hi
As you aware or might be not, [the latest JDK11 11.0.11_9 update](https://bugs.openjdk.java.net/browse/JDK-8202343) depreciated TLSv1.0 and TLSv1.1

These causing serious problems with connection to old MySQL databases (v 5.6, v5.7)
with exception something like:
```
javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)
```

It affected liquibase docker too.
for example, we were using docker of the latest liqubase but a few days ago (after global JDK11 update)) it suddenly stop working because your parent image is not locked to a specific JDK. you have:
`FROM openjdk:11-jre-slim-buster`

However, should be:
`FROM openjdk:11.0.XX-jre-slim-buster`

to be more specific:
`FROM openjdk:11.0.10-jre-slim-buster`
11.0.10 - was the last JDK11 version without TLS deprecation.

So in other words you locked liquibase version, but you didn't lock JDK version in your parent image and that causing problems now.
Also if you will lock JDK version, your tagging system also will have to be improved, as you for example might have:
liquibase:4.2 but with JDK11.0.10 and the same liquibase version with the latest JDK

For now as a temp solution, we downgraded to the older version of liquibase

**Comment by r2liquibase** [comment]
Understood. We would like to stay on the latest JDK version. 

Our Dockerfile is available in the repos. Perhaps fork the repos and make the JDK change to just work with older MySQL...?

Please let us know your thoughts!

**Comment by Bryksin** [comment]
Hi

>We would like to stay on the latest JDK version.

Yes, I'm totally with you, liquibase should be using the latest JDK, but... it should not affect past releases.
If liquibase 4.1 was released with JDK11.0.8 (for example, as I'm not sure what version it was) it should remain with it even after a year+. Potentially you could release an update and make an old version to work with newer JDK
For example, today when u have already liquibase 4.3+, you could still release an update to docker image v4.1 with newer JDK but the old one also should remain available.

To achieve so I see only one option: improve your tagging naming convention, instead of just stating liquibase version include also JDK version.

Another approach and even simpler one - never update old releases, so if your old release of liquibase 4.1 was using JDK 11.0.8 it should never get updated, so if I'm using liquibase 4.1 it will always remain with the locked JDK version. whenever u would release the next liquibase update you can update JDK with it, so 4.1 will remains forever with JDK11.0.8 and since liquibase 4.3+ JDK will be 11.0.11 forever.

But now your JDK version is not locked and such updates as recent affected a lot of your users, even though they didn't change anything. Yesterday docker image was working, but today after pulling it parent layer was automatically replaced with a newer parent, as the parent image version was not locked and it suddenly stopped working for everyone.
